paf-mvp-pattern-library-audit/stories/components/AdvertImage.js

```diff
@@ -0,0 +1,57 @@
+import Details from './Details';
+import { BlockedImage } from './Icons';
+
+const Image = (args = {}) => {
```

Comment:
Why not using Typescript? (https://storybook.js.org/docs/react/configure/typescript)

Reply:
The pattern library is produced to be final implementation agnostic. These will be moved to typescript when the HTML and CSS is transferred for the reference implementation. Other optimisations will also be applied to reduce duplications, etc. The advantage of the approach is that someone could take this HTML and JS and deploy it in PHP (or another language) without any issues.

Reply:
Thanks.
I'm sorry, I don't understand this sentence:
> These will be moved to typescript when the HTML and CSS is transferred for the reference implementation

Could you please develop? What do you mean by "transferred for the reference implementation"?

Reply:
My understanding is that it's pretty common to define components and stories in typescript, isn't it?
https://storybook.js.org/docs/react/get-started/whats-a-story